### PR TITLE
Update license and add a basic contribution guide

### PR DESCRIPTION
**Pull request by MGaetan89**
As discussed in #326, this PR unify the project license to use MIT.

I've also added a basic contribution guide. The extra suggestions in https://github.com/evrencoskun/TableView/issues/326#issuecomment-763025357 can be addressed in a separate PR, but adding Pull Request and Issues templates to the project.

### CODE REVIEW

File: CONTRIBUTORS.md [hunk 1]
```diff
@@ -1,20 +0,0 @@
-TableView contributors (sorted alphabetically)
```
Comment (MGaetan89) [inline CONTRIBUTORS.md:1]: I've replaced this file with a link to GitHub's [contributors' list](https://github.com/evrencoskun/TableView/graphs/contributors), so there no need to manually maintain a file. Let me know if you prefer to keep this file.

File: README.md [hunk 2]
```diff
@@ -53,26 +53,14 @@ dependencies {
...
 ## Sample Apps
 
-- This repository has an Sample Application of TableView.
+- This repository has a [sample application](https://github.com/evrencoskun/TableView/tree/master/app) of `TableView`.
 - [TableViewSample 2](https://github.com/evrencoskun/TableViewSample2)
 - [Xently-UI](https://github.com/ajharry69/Xently-UI)
-- (contact me to mention your app on this page)
-
-
-## Communication
```
Comment (MGaetan89) [inline README.md:67]: All this section has moved to the `CONTRIBUTING.md` file

File: CONTRIBUTORS.md (hunk 1 above)
Comment (Zardozz) [reply CONTRIBUTORS.md:1]: My only comment is that GitHub's contributors' list, only covers direct contributions and not those pulled in from forks.

**Comment by evrencoskun** [comment]
Looks fine Thanks @MGaetan89

**Pull request merged**